examples/hat/binaryclock.py

```diff
@@ -0,0 +1,35 @@
+import unicornhat as uh
```

Comment:
Very first line should be `#!/usr/bin/env python` - for better or worse we use this convention across all our Python examples so they can be run with: `./examplename.py`